nanshe_ipython.ipynb

```diff
@@ -1173,7 +1173,25 @@
         "                da_imgs_flt.dtype.itemsize >= 4):\n",
         "            da_imgs_flt = da_imgs_flt.astype(np.float32)\n",
         "\n",
-        "        da_result = normalize_data(da_imgs)\n",
+        "        da_imgs_flt_mins = da_imgs_flt.min(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "        da_imgs_flt_maxes = da_imgs_flt.max(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "\n",
+        "        da_imgs_flt = (\n",
+        "            (da_imgs_flt - da_imgs_flt_mins) /\n",
+        "            (da_imgs_flt_maxes - da_imgs_flt_mins)\n",
```

Comment:
This rescaling isn't strictly necessary as it will end up being cancelled out by normalization. Though there is no harm in having it here either.